files/build_templates/dhcp_relay.service.j2

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=DHCP relay container
-Requires=updategraph.service swss.service teamd.service
-After=updategraph.service swss.service teamd.service
+Requires=updategraph.service swss.service syncd.service teamd.service
```

Comment:
Why is Requires needed? Is After not enough?

Reply:
`Requires` will start the service ONLY if the dependency is active and has successfully started. `After` is more of an order-only dependency and will start the service after the dependency should start, regardless of if it is enabled or successfully started.

Reply:
I understand how Requires work. That wasn't my question.

My question was that why did syncd added to required for dhcp_relay but not any other services below. Can you explain?

Reply:
This wasn't my intention. This change was made by mistake. I reverted it.
This PR is only about order